tests/compile_tests/font/test.bas

```diff
@@ -1,8 +1,45 @@
+OPTION _EXPLICIT
 $CONSOLE:ONLY
 CHDIR _STARTDIR$
 
-Dim OldFont AS LONG
-OldFont = _LOADFONT("slick.ttf", 10)
+DIM fh AS LONG
+
+PRINT "Loading font using filename...";
+fh = _LOADFONT("slick.ttf", 10)
+
+IF fh > 0 THEN
+    PRINT "done."
+    PRINT "Font height ="; _FONTHEIGHT(fh)
+    _FREEFONT fh
+ELSE
+    PRINT "failed!"
+END IF
+
+
+PRINT "Loading font from memory...";
+fh = _LOADFONT(LoadFileFromDisk("slick.ttf"), 20, "memory")
+
+IF fh > 0 THEN
+    PRINT "done."
+    PRINT "Font height ="; _FONTHEIGHT(fh)
+    _FREEFONT fh
+ELSE
+    PRINT "failed!"
+END IF
+
 
-print OldFont;
 SYSTEM
+
+' Loads a whole file from disk into memory
+FUNCTION LoadFileFromDisk$ (path AS STRING)
+    IF _FILEEXISTS(path) THEN
+        DIM AS LONG fh: fh = FREEFILE
+
+        OPEN path FOR BINARY ACCESS READ AS fh
+
+        LoadFileFromDisk = INPUT$(LOF(fh), fh)
```

Comment:
Huh, did not know you could use `INPUT$()` like that. I always do a silly `SPACE$()` + `GET #f` combo to read whole files 🤦

Reply:
It depends, AFAIK `INPUT$` if used that way, stops reading when hitting a EOT marker (CTRL-Z (ascii 26)), so `SPACE$` + GET is the safer method if you don't know the contents of a file and will be definitly the required way to go for binary files.